python_modules/dagster/dagster/core/types/evaluator.py

```diff
@@ -163,6 +163,12 @@ def friendly_string_for_error(error):
             path_msg=path_msg,
             type_msg=print_config_type_to_string(type_in_context, with_lines=False),
         )
+    elif error.reason == DagsterEvaluationErrorReason.MISSING_REQUIRED_FIELDS:
+        return 'Missing required fields "{field_names}" {path_msg} Expected: "{type_msg}"'.format(
```

Comment:
so i think we can just start returning error.message for these and totally remove the "friendly" string, which is now redudant

Reply:
is that right? i just hit the "not supported" error message which was really bad so i updated this.

Reply:
I believe that error is high quality and i believe it is also the one rendered in dagit. this path is only hit in the unittest API? It's a bit confused atm. We definitely need the if error.reason check. I just think we can return error.message directly

Reply:
to clarify i was talking about hitting [1] since DagsterEvaluationErrorReason.MISSING_REQUIRED_FIELDS was not declared here

Reply:
yes you should definitely add that clause. all i'm saying is that you return error.message instead of constructing a new error string. all the "friendly" error messages are duplicative at this point and vestiges of a previous structure that left these error messages in place

Reply:
got it, i was able to repro the condition i hit and verify error.message is reasonable